benchmarks/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+add_executable(inexor-vulkan-renderer-benchmarks
+    engine_benchmark_main.cpp)
+
+set_target_properties(inexor-vulkan-renderer-benchmarks PROPERTIES
+    CXX_EXTENSIONS OFF
+    CXX_STANDARD 17
+    CXX_STANDARD_REQUIRED ON)
```

Comment:
Can you put all closting brackets in a new line? to stay consistent

Reply:
This conforms to the cmake style guide.